### PR TITLE
[Serverless] add serverless CLI, support subcmd: deploy, info, invoke

### PR DESCRIPTION
**Pull request by deathemperor**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the current behavior? (You can also link to an open issue here)**
Serverless not supported

**What is the new behavior (if this is a feature change)?**
Serverless supported

**Additional info:**
I added all subcommands and worked basic suggestions for deploy, info, invoke subcommand. Hopefully I can improve it with color and value suggestions, this is my first PR on this repo.

Hopefully can finish all about `serverless` CLI!

### CODE REVIEW

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks a lot for your first contribution! I left some comments, mostly for suggestions and templates.

File: dev/serverless.ts [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+    args: {
+      name: "function",
+    },
+  },
+  region: {
+    name: ["--region", "-r"],
+    description: "Region of the service",
+    args: {
+      name: "us-east-1",
+    },
```
[the remark this answers is not in the document]
Comment (deathemperor) [reply dev/serverless.ts:23]: agree! such feature is what I need to learn more on how to do it with fig. Will add that.

File: dev/serverless.ts [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+    name: "--use-local-credentials",
+    description:
+      "Rely on locally resolved AWS credentials instead of loading them from Dashboard provider settings (applies only to services integrated with Dashboard)",
+  },
+  config: {
+    name: ["--config", "-c"],
+    description: "Path to serverless config file",
+    args: {
+      name: ".yml file",
+    },
```
[the remark this answers is not in the document]
Comment (deathemperor) [reply dev/serverless.ts:35]: sure

**Comment by QuiiBz** [comment]
Hi, I resolved the review and added suggestions for the regions.

**Review by QuiiBz — APPROVED** [review]

**Comment by deathemperor** [comment]
awesome! thanks! I'll add more options when I have time

**Pull request merged**